### PR TITLE
feat: create function to send mail for action export download link

### LINKED ISSUE #17: Create function to send mail for action export

### PR DESCRIPTION
**Pull request by pyphilia**
close #17

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -107,9 +114,36 @@ const plugin: FastifyPluginAsync<MailerOptions> = async (fastify, options) => {
...
+      days: expirationDays,
+    });
+
+    const html = await fastify.view(`${modulePath}/templates/exportActions.eta`, {
+      member,
+      link,
+      translated,
+      information,
+    });
+    await sendMail(fromEmail, member.email, 'Analytic Traces Download Link', link, html);
```
Comment (spaenleh) [inline src/index.ts]: I see the other ones are also hard-coded but maybe we could have the Subject as a constant somewhere ?

File: src/lang/en.json [hunk 2]
```diff
@@ -5,5 +5,7 @@
     "unrequestedregister": "If you don't remember signing up to Graasp, you can just ignore this email.",
     "signin": "Sign In",
     "magiclink": "You asked us to send you a magic link for quickly signing in to your account. Here your go:",
-    "unrequestedlink": "If you haven't requested the link, no further action is required. It will expire soon, don't worry you account and data are safe."
+    "unrequestedlink": "If you haven't requested the link, no further action is required. It will expire soon, don't worry you account and data are safe.",
```
Comment (spaenleh) [inline src/lang/en.json]: ... don't worry `your` account ...

**Review by spaenleh — CHANGES_REQUESTED** [review]
Looks good, just a few comments 🍭

File: src/index.ts (hunk 1 above)
Comment (pyphilia) [reply src/index.ts]: I used this opportunity to also translate the mail titles!

**Review by spaenleh — APPROVED** [review]
🍕 Yes ! Nice for the translations !

**Pull request merged**